packages/third-parties/agenda/src/AgendaModule.spec.ts

```
@@ -89,17 +91,17 @@ describe("AgendaModule", () => {
         const agendaModule = PlatformTest.get<any>(AgendaModule)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)